src/donejs-carousel-plugin.stache

```diff
@@ -30,7 +30,7 @@
 
 {{#if ./isDesktopBrowser}}
 	
-	{{#is ./slides.length 1}}
+	{{#if ./isLengthOneOrLess(./slides.length)}}
```

Comment:
this means that now if there are no slides being passed in, the carousel div will still render but it'll be blank but probably still take up some space. before, it would be as if the carousel didn't exist. is this desired behaviour?

Reply:
Before if no slides were passed in, it would render the full carousel (the else block). But I see your point. Actually this function isn't even necessary, because I am accounting for 0 slide scenario in the parent stache, displaying none on the carousel component if no slides and showing 'no slide' message. I'm going to revert it back to the previous version and assume the user will account for 0 slide scenario (for now).

Reply:
alright cool 👍